app/src/features/charts/DisparityBarChart.tsx

```diff
@@ -102,7 +108,7 @@ function getSpec(
             x: { scale: "x", field: thickMeasure },
             y: { scale: "y", field: breakdownVar, band: 0.8 },
             text: {
-              signal: `format(datum["${thickMeasure}"], "") + "%"`,
+              signal: `isValid(datum["${thickMeasure}"]) ? datum["${thickMeasure}"] + "%" : "" `,
```

Comment:
Can you add the isValid check to the simple bar chart too? I see NaN there for unknown